packages/flutter/test/material/text_field_test.dart

```diff
@@ -1208,7 +1277,7 @@ void main() {
       editable.getLocalRectForCaret(const TextPosition(offset: 0)).topLeft,
     );
 
-    expect(topLeft.dx, equals(399));
+    expect(topLeft.dx, equals(398.5));
```

Comment:
Why did the origin of these tests change? Is the text field's new default cursor width changing TextField's instrinsic width?  I didn't think we expected this PR to introduce any layout changes.

Reply:
The new width (1.o -> 2.0) is causing this change.

Reply:
Does it only affect empty TextFields?

Reply:
It looks like it affects textfields with text in them, but the test shows that entering text does not change the layout.

Reply:
Sorry I misread this test change: I was assuming that we were checking the TextField's topLeft, not the caret's topLeft. The caret edge moving over a half pixel totally makes sense. Sorry about the confusion.